### ISSUE TITLE
Q: Pixel averaging for kymo generation

### ISSUE THREAD

**Issue by clairelbromley** [dougkelly88/ClaireDataAnalysis#92]
Will a change in pixel size alter the optimal number of pixels to average over for each kymograph?

**Comment by dougkelly88** [comment]
Yes, insofar as kymograph spacing along the cut axis is defined in um and extent to average over per kymograph is defined in pixels.
